ote/src/clj/ote/services/transit_visualization.sql

```diff
@@ -170,3 +170,22 @@ SELECT p.id, p.created,
   JOIN "external-interface-description" eid ON p."external-interface-description-id" = eid.id
  WHERE p."transport-service-id" = :service-id AND p."deleted?" = FALSE
  ORDER BY p.created DESC;
+
+-- name: detected-changes-for-date
+SELECT c.date, c."added-routes", c."removed-routes", c."changed-routes", c."no-traffic-routes", c."current-week-date",
+       c."different-week-date", c."change-date", c.created, c."transport-service-id"
+FROM "gtfs-transit-changes" c
+WHERE c."transport-service-id" = :service-id
+  AND c.date = :date::DATE;
+
+-- name: detected-route-changes-for-date
+SELECT r."route-short-name", r."route-long-name", r."trip-headsign",
```

Comment:
Better name: changed-routes-to-service-by-date

Reply:
Roger